components/layout_2020/display_list/background.rs

```diff
@@ -167,7 +167,7 @@ pub(super) fn layout_layer(
     painter: &BackgroundPainter,
     builder: &mut super::DisplayListBuilder,
     layer_index: usize,
-    intrinsic: IntrinsicSizes,
+    intrinsic: NaturalSizes,
```

Comment:
```suggestion
    intrinsic: NaturalSizes,
```
    natural: NaturalSizes,
```

Reply:
Okay. I've done this rename and more for consistency.